packages/sync/src/modules/class-users.php

```diff
@@ -643,6 +660,20 @@ public function estimate_full_sync_actions( $config ) {
 		return (int) ceil( $count / self::ARRAY_CHUNK_SIZE );
 	}
 
+	/**
+	 * Return Total number of objects.
+	 *
+	 * @param array $config Full Sync config.
+	 *
+	 * @return int total
+	 */
+	public function total( $config ) {
+		global $wpdb;
+		$where = $this->get_where_sql( $config );
+		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		return $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->usermeta WHERE $where" );
```

Comment:
Did you mean `$wpdb->users`? Feels odd to be calculating totals on behalf of the users module by using the usermeta table. Or perhaps I'm misunderstanding what's the end goal here?

Reply:
if you look at the `$where` clause you can use that it filters by things on the usermeta table, privileges, etc.

Reply:
Good point. 

But since we changed the table name to be `usermeta` here, can't we just use the `total()` method from the parent class here?